share/spice/symbolab/subtitle_content.handlebars

```diff
@@ -0,0 +1 @@
+Solution for: <span {{#if problemHasLatex}}class="mathquill-embedded-latex"{{/if}}>{{problem}}</span>
```

Comment:
It looks like we need to wrap this in an `<h4>` to ensure we apply the correct subtitle styling to this:

```handlebars
<h4 class="c-base__sub">
     Solution for: <span {{#if problemHasLatex}}class="mathquill-embedded-latex"{{/if}}>{{problem}}</span> 
</h4>
```